chemie/corporate/templates/corporate/list_companies.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Bedrifter</title>
+</head>
+<body>
+
+{% if companies %}
+
+<ul>
+    {% for company in companies %}
+    <li>{{ company.name }}</li>
+    {% endfor %}
+</ul>
```

Comment:
Dette "if companies ... for company ... Kan egentlig byttes ut med [for … empty](https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#for-empty)